### PR TITLE
Address #396 to alleviate the recurring cachedir removal problem

### PR DESCRIPTION
**Pull request by oozie**
To reproduce:
* pull up a graph in TSD,
* `rm -fr /tmp/opentsdb`
* go back to the browser, press `F5`

With this change, the graph will load after `F5` is pressed.

### CODE REVIEW

File: test/graph/TestPlot.java [hunk 1]
```diff
@@ -0,0 +1,97 @@
+// This file is part of OpenTSDB.
```
Comment (manolama) [inline test/graph/TestPlot.java]: Make sure to add this file to the Makefile so the sh script can test it

**Comment by manolama** [comment]
Looks good. It might be worth it to move the ``checkDirectory()`` method to a Utility class like ``FileUtility.java``.

**Comment by oozie** [comment]
Thanks for the review - suggestions applied.

**Comment by manolama** [comment]
Merged in 8f06f6ecefefc8d187c3d29b81fa62fe4d0d2169

**Pull request closed without merging**